pkg/deploy/api/types.go

```diff
@@ -115,6 +115,9 @@ type RollingDeploymentStrategyParams struct {
 	// TimeoutSeconds is the time to wait for updates before giving up. If the
 	// value is nil, a default will be used.
 	TimeoutSeconds *int64
+	// UpdatePercent is the percentage of replicas to scale up or down each
+	// interval. If nil, one replica will be scaled up and down each interval.
```

Comment:
I probably missed other discussion, but is one the most reasonable default?  It seems a little weird to specify a percentage, but if you don't specify a percentage, we don't choose a default percentage, we choose an absolute value.